tests/check_check_master.c

```diff
@@ -104,6 +104,22 @@ static master_test_t master_tests[] = {
   { "Simple Tests", CK_FAILURE, "Assertion 'x >= y' failed: x == 2, y == 3" },
   { "Simple Tests", CK_FAILURE, "Assertion '1%d >= 3%f' failed: 1%d == 0, 3%f == 1" },
   { "Simple Tests", CK_PASS,    "Passed" },
+  /* Tests on floating macros */
+  { "Simple Tests", CK_FAILURE, "Assertion 'x == y' failed: x == 1.100000, y == 1.200000" },
```

Comment:
You will need to rebase against master. I added a change today which will add an entry to this table. Now one will need to include the name of the method under test. For example, the first entry will now become:

{ "Simple Tests", "test_ck_assert_floating_eq", CK_FAILURE, "Assertion 'x == y' failed: x == 1.100000, y == 1.200000" }